app/models/card.rb

```diff
@@ -11,21 +11,55 @@ class Card < ActiveRecord::Base
   validates :original_text, :translated_text, :deck_id, presence: true
   validate :original_and_translated_not_equal
 
-  before_create :set_review_date
+  before_create :set_default_attributes
 
   def check_translation(translation)
     if translated_text == translation
-      update_attribute(:review_date, Time.now + 3.days)
-      return true
+      translation_is_correct
     else
-      return false
+      translation_is_incorrect
     end
   end
 
   private
 
-  def set_review_date
+  def translation_is_correct
```

Comment:
Assignment Branch Condition size for translation_is_correct is too high. [26.68/15]